reference/5.1/Microsoft.PowerShell.Core/About/about_Logging.md

```diff
@@ -0,0 +1,139 @@
+The associated ETW provider `GUID` is `A0C1853B-5C40-4B15-8766-3CF1C58F985A}`.
+
+When script block logging is enabled,
+PowerShell will log the following events to the `Microsoft-Windows-PowerShell/Operational` log:
+
+|Field| Value|
+|-|-|
+|EventId|`4104` / `0x1008`|
+|Channel|`Operational`|
+|Level|`Verbose`|
+|Opcode|`Create`|
+|Task|`CommandStart`|
+|Keyword|`Runspace`|
+
+## Enabling Script Block Logging
+
+Any new PowerShell session will pickup the new setting after making one of the following changes.
+
+> [!NOTE]
+> It is recommended to enable Protected Event Logging when
+> using Script Block logging for anything other than diagnostics purposes.
+
+### Using Group Policy
+
+To enable automatic transcription,
+enable the ‘Turn on PowerShell Script Block Logging’
+feature in Group Policy through `Windows Components -> Administrative Templates -> Windows PowerShell`.
+
+### Using the Registry
+
+Run the following function:
+
+```powershell
+function Enable-PSScriptBlockLogging
+{
+    $basePath = “HKLM:\Software\Policies\Microsoft\Windows\PowerShell\ScriptBlockLogging”
+
+    if(-not (Test-Path $basePath))
+    {
+        $null = New-Item $basePath -Force
+    }
+
+    Set-ItemProperty $basePath -Name EnableScriptBlockLogging -Value “1”
+}
+```
+
+## Protected Event Logging
+
+One concern when increasing the amount of logging on a system
+is the danger that logged content may contain sensitive data.
+For example, if you log the content of every PowerShell script that was run,
+there is the possibility that a script may contain credentials or other sensitive data.
+
+If an attacker later compromises a machine that has logged this data,
+it may provide them with additional information with which to extend their reach.
+
+To prevent this dilemma,
+Windows 10 introduces Protected Event Logging.
+Protected Event Logging lets participating applications encrypt
+sensitive data as they write it to the event log.
+You can then decrypt and process these logs once you’ve
+moved them to a more secure and centralized log collector.
+
+Protected Event Logging protects event log content through the
+IETF Cryptographic Message Syntax (CMS) standard.
+The CMS encryption standard implements public key cryptography,
+where the keys used to encrypt content (the public key)
+and the keys used to decrypt content (the private key) are separate.
+
+Your public key can be shared widely,
+and is not sensitive data.
+If any content is encrypted with this public key,
+only your private key can decrypt it.
+For more information about Public Key Cryptography, see [Wikipedia - Public Key Cryptography](http://en.wikipedia.org/wiki/Public-key_cryptography).
+
+When you implement a protected event logging policy,
+you deploy a public key to all machines that have event log data you want to protect.
+You retain the corresponding private key to post-process the event logs at a more secure location
+such as a central event log collector, or SIEM aggregator.
```

Comment:
What is 'SIEM'?

Reply:
Add a link... this is a generic term is security for thing like log analytics.